server/services/config/crm7DisplayConfig.json

```diff
@@ -116,23 +116,6 @@
             {
               "label": "Matter type",
               "apiField": "summary.matterType"
-            }
-          ]
-        },
-        {
-          "title": "Equal Opportunities",
```

Comment:
Are you sure that these fields should be removed? As they are in the mappings spreadsheet and there is also a screenshot showing these fields.

Reply:
Yes, I had a chat with both `Sheikh` and `Prince`, they confirmed that, these fields can be removed.